pyke/targetpixelfile.py

```diff
@@ -226,7 +244,10 @@ def to_fits(self):
         raise NotImplementedError
 
     def _get_aperture_flux(self):
-        return np.nansum(self.flux[:, self.aperture_mask], axis=1)
+        af = np.nansum(self.flux[:, self.aperture_mask], axis=1)
+        npix = np.sum(self.aperture_mask)
+        er = (1./npix)*(np.nansum(self.flux_err[:, self.aperture_mask]**2, axis=1)**0.5)
```

Comment:
I would suggest:
```
er = np.sqrt(np.nansum(self.flux_err[:, self.aperture_mask]**2, axis=1)) / pix
```
just for the sake of readability

Reply:
Could you add a unit test for this?

Reply:
Unit tests will come when both PRs are in, it'll be easier for me to write them when we can have both functionalities.

Reply:
A nice new unit test has been added for this.